src/renderer/components/message/Message.tsx

```diff
@@ -819,3 +828,70 @@ function WebxdcMessageContent({ message }: { message: T.Message }) {
     </div>
   )
 }
+
+/**
+ * displays a VCard attachement with avatar, mail & name
+ * onClick imports VCard and creates chat
+ * (only first if multiple contacts are included in VCard)
+ */
+function VCardContent(message: T.Message) {
+  const { selectChat } = useChat()
+  const accountId = selectedAccountId()
+  const { openDialog } = useDialog()
+  const tx = useTranslationFunction()
+  if (message.vcardContact) {
+    const { profileImage, color, displayName, addr } = message.vcardContact
+    const startChatWithContact = async (addr: string) => {
+      const continueProcess = await new Promise((resolve, _reject) => {
+        openDialog(ConfirmationDialog, {
+          message: tx('ask_start_chat_with', displayName),
+          confirmLabel: tx('ok'),
+          cb: resolve,
+        })
+      })
+
+      if (!continueProcess) {
+        return null
+      }
+
+      const contactIds = await BackendRemote.rpc.importVcard(
+        accountId,
+        message.file ?? '' // vCard messages always have a file property
```

Comment:
```suggestion
        message.file! // vCard messages always have a file property
```
would also be possible

Reply:
but does not matter probably, your solution ensures that it is a string, that's probably better